### ISSUE TITLE
noisy inference, any idea why ?

### ISSUE THREAD

**Issue by YacineDeghaies** [prs-eth/Marigold#106]
After training the model using my own dataset and replacing the unet/ folder, the inference seems to be very slow is there a way to improve its speed ?  

For training I used the stable-diffusion-2

The inferred images look so noisy too, any idea why ? 

thank you !
inferred image:
![output_image](https://github.com/user-attachments/assets/6a05cabb-e94c-41be-a724-2166edcca4ba)

Input images:
![shot_0003_source_0000](https://github.com/user-attachments/assets/e78c3290-9cd7-4133-b1e6-6d310fde7923)

ground truth depth map:
![shot_0003_dm_gt_s_0000](https://github.com/user-attachments/assets/f7edf05a-5fa6-4974-86a5-739e2fce339f)

The inference setting:
```

#!/usr/bin/env bash
set -e
set -x

# Use specified checkpoint path, otherwise, default value
ckpt=${1:-"checkpoint/stable-diffusion-2"}
subfolder=${2:-"eval"}
BASE_DATA_DIR="/vol/fob-vol3/mi20/deghaisa/code/"

python infer.py  \
    --checkpoint $ckpt \
    --seed 1234 \
    --base_data_dir $BASE_DATA_DIR \
    --denoise_steps 50 \
    --ensemble_size 10 \
    --processing_res 0 \
    --dataset_config config/dataset/data_dsb_test.yaml \
    --output_dir output/${subfolder}/dsb_test/prediction \

```

**Comment by markkua** [comment]
Hi, you can use smaller ensemble size and less steps, for example, 10 steps with enseomble_size = 1

**Comment by YacineDeghaies** [comment]
> Hi, you can use smaller ensemble size and less steps, for example, 10 steps with enseomble_size = 1

By less steps do you mean --denoise_steps 10 ?

**Comment by markkua** [comment]
yes

**Comment by YacineDeghaies** [comment]
My Depth Maps have 1-channel. They are 8-bit relative depth maps. Can this be also a factor ?

**Comment by markkua** [comment]
If you are talking about the noise on your prediction, I recommend to try it with our checkpoint, and also consider if your training data is diverse enough or if the model was trained for enough steps.
8-bit depth maps have worse accuracy compared to regular depth datasets. This could be one reason.

**Comment by YacineDeghaies** [comment]
> I recommend to try it with our checkpoint

For training I used stable-diffusion-2 (downloaded from HuggingFace)
Do you mean I should use marigold-depth-lcm-v1-0 for inference ? If I'm correct I just need then to replace its unet/ with my trained unet/ right ? I hope am not confused haha

Thank you for baring with me ...
